From b0639eca089256c1d816b62c0e2b901cc14a5f7c Mon Sep 17 00:00:00 2001
From: Michael Orlitzky <michael@orlitzky.com>
Date: Wed, 28 Aug 2024 10:57:17 -0400
Subject: [PATCH] support/configure.pl: disable perl version check

The test suite apparently passes with perl-5.40, and that's what
we have in ~arch on Gentoo, so let's accept it.
---
 support/configure.pl | 37 -------------------------------------
 1 file changed, 37 deletions(-)

diff --git a/support/configure.pl b/support/configure.pl
index 5615c65..95f66de 100644
--- a/support/configure.pl
+++ b/support/configure.pl
@@ -14,43 +14,6 @@
 #  GNU General Public License for more details.
 #-------------------------------------------------------------------------------
 
-BEGIN {
-   if ($] < 5.016) {
-      print STDERR <<".";
-polymake requires perl version not lower than 5.16;
-your perl interpreter says it is $].
-
-Please upgrade your perl installation;
-if you already have an up-to-date perl interpreter somewhere else,
-you can specify its location on the command line:
-
-./configure PERL=/path/to/my/new/perl [other options ...]
-.
-      exit(1);
-   } elsif ($] >= 5.040) {
-      print STDERR <<".";
-*************
-*** ERROR ***
-*************
-
-polymake has not been checkced for compatibility with perl 5.40 or newer;
-your perl interpreter says it is $].
-
-If you already have another (older) perl interpreter somewhere else, you can
-specify its location on the command line:
-
-./configure PERL=/path/to/my/new/perl [other options ...]
-
-You can install a custom perl version in your home directory using perlbrew
-(https://perlbrew.pl). Make sure to append '-Duseshrplib' to the install
-command if you want to use libpolymake (e.g. for the jupyter interface) and to
-install all required perl modules which are listed after running configure
-again with the new perl.
-.
-      exit(1);
-   }
-}
-
 use Config;
 use Cwd;
 use File::Path;
-- 
2.46.0

